juniper_official/routing/check-te-rsvp-interface-errors.rule

```diff
@@ -218,7 +218,19 @@ healthbot {
                         }
                         next;
                     }
+                }					
+                term no-authentication-fail {
```

Comment:
What if authentication-fail is not transmitted?

Reply:
Then due to zero-suppression in the rule, it will match 0 and go to default condition of green.

Reply:
Added "when exists" for all green conditions.